pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -7,7 +7,6 @@ import (
 	"sync"
 	"sync/atomic"
 
-	"github.com/hashicorp/go-multierror"
```

Comment:
https://github.com/hashicorp/go-multierror/issues/98

Reply:
They're functionally similar but have differing output so it's not a 1-1 replacement. So lets be careful!

Reply:
Yes, I noticed that when I was writing the test, multierror says "2 errors occurred: x y". I don't have a preference in terms of formatting, which is why I chose `errors.Join`